lib/rules/no-irregular-whitespace.js

```diff
@@ -18,13 +18,20 @@ module.exports = function(context) {
     // Module store of errors that we have found
     var errors = [];
 
+    // Comment nodes.  We accumulate these as we go, so we can be sure to trigger them after the whole `Program` entity is parsed, even for top-of-file comments.
+    var commentNodes = [];
+
+    // Lookup the `skipComments` option, which defaults to `true`.
+    var options = context.options[0] || {};
+    var skipComments = typeof options.skipComments === "undefined" ? true : !!options.skipComments;
```

Comment:
We didn't have this option before so I think making it "on" by default is a breaking change, thus a bad idea.